GenderPayGap.WebUI/Views/AccountCreation/AlreadyCreatedAnAccountQuestion.cshtml

```diff
@@ -76,15 +76,18 @@
                 })
 
                 <div class="govuk-inset-text">
-                    You will need to create your own user account even if someone else has previously reported your
-                    organisation's gender pay gap.
+                    You will need to create your own user account even if someone else has
+                    previously reported your employer’s gender pay gap.
                 </div>
 
                 @(Html.GovUkButton(new ButtonViewModel
                 {
                     Text = "Continue"
                 }))
+                @{ var reportingUrl = Url.Action("ManageOrganisationsGet", "ManageOrganisations"); }
+                <a href="@(Global.StartUrl ?? reportingUrl)" class="govuk-button" role="button">Cancel</a>
```

Comment:
We use links for 'Cancel' buttons, but we always style them as links rather than buttons. 
The pattern can be seen in `ChangeContactPreferences.cshtml` on lines 68-72. We need to surround links with `p` body styling so the text is formatted correctly - particularly so the font size is inherited.